lib/Twig/MainExtension.php

```diff
@@ -117,11 +119,13 @@ private function getAssetCacheBuster(string $path, string $rootPath): string
         return substr(sha1($contents), 0, 6);
     }
 
-    public function yamlEncode(mixed $value)
+    public function yamlEncode(mixed $value): mixed
     {
         if (is_string($value)) {
             return str_replace('\\', '\\\\', $value);
         }
+
+        return $value;
```

Comment:
How did this work in the past when return wasn't present before? Is this method even called?